src/ark/phenotyping/cluster_helpers.py

```diff
@@ -140,12 +147,12 @@ def generate_som_clusters(self, external_data: pd.DataFrame) -> np.ndarray:
 
         # work in batches of 100 to account to support large dataframe sizes
```

Comment:
A verrrrry small comment - but it'd be good to update this comment here so it doesn't say "batches of 100"